### ISSUE TITLE
[Bug]: False Positive

### ISSUE THREAD

**Issue by admgihub** [CyberDrain/Check#124]
### Required confirmations before submitting

- [x] **I can reproduce this issue on the latest released version** of Check.
- [x] **I have searched existing issues** (both open and closed) to avoid duplicates.
- [x] I am **not** requesting general support; this is an actual bug report.

### Issue Description

Please could URL: https://lifecycle.ifs.com/* be whitelisted in the json rule detection? It would be greatly appreciated

### Extension Version

1.1.0

### Rules Version

1.1.0

### Relevant Logs / Stack Trace

```plaintext
Attempted to apply via powershell script and policy but receive warning
{
  "exclusion_system": {
    "domain_patterns": [
      "^https://lifecycle\\.ifs\\.com(/.*)?$"
    ]
  }
}
```
